README.md

```diff
@@ -1,4 +1,35 @@
-# iroha2-block-explorer-backend
+# Iroha 2 Blockchain Explorer Backend
+
+This readme provides an overview of the Iroha 2 blockchain explorer backend and instructions on how to install, run, and utilize the explorer's backend features.
+
+## Installation
+
+To set up the Iroha 2 blockchain explorer backend, follow these steps:
+
+1. Install Rust.
+2. For full functionality of the explorer backend, build [Iroha `v2.0.0-pre-rc.9`](https://github.com/hyperledger/iroha/tree/ea45b5053018acd48340024800786ff5a3d0904d) and ensure it is running.
+
+3. Build the explorer backend binary by running the following command:
+
+```bash
+cargo build --release 
+```
+
+4. Prepare Iroha Client config ([reference](https://github.com/hyperledger/iroha/blob/ea45b5053018acd48340024800786ff5a3d0904d/docs/source/references/config.md)). **Define target peer location here**.
+Or copy the configuration file from  [explorer-deploy-dev-tool](https://github.com/0x009922/explorer-deploy-dev-tool) Config Files
```

Comment:
It's not clear what should be done in this step. You probably mean create a peer configuration file? Current phrasing is a bit odd.
You can also refer to the Iroha docs for this: https://hyperledger.github.io/iroha-2-docs/guide/configure/peer-configuration.html